src/lib.rs

```diff
@@ -145,7 +148,7 @@ impl FragmentScanner {
                 if !meta.file_type().is_file() {
                     if let Ok(target) = fs::read_link(&fpath) {
                         // A devnull symlink is a special case to ignore previous file-names.
-                        if target == PathBuf::from("/dev/null") {
+                        if target.as_path() == Path::new(DEVNULL) {
```

Comment:
You don't need as_path() here, `PathBuf == Path` exists

Reply:
Thanks, fixed!